<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="oneButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>One</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Two</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threeButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Three</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Four</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fiveButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Five</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
